sdk/src/main/java/io/opentelemetry/sdk/trace/TracerSdk.java

```diff
@@ -77,6 +77,17 @@ public void recordSpanData(SpanData span) {}
     return null;
   }
 
+  /**
+   * Attempts to stop all the activity for this {@link Tracer}. Calls
+   * {@link SpanInterceptor#shutdown()} for all registered {@link SpanInterceptor}s.
+   *
+   * <p>This operation may block until all the Spans are processed. Must be called before turning
+   * off the main application to ensure all data are processed and exported.
+   *
+   * <p>After this is called all the newly created {@code Span}s will be no-op.
+   */
+  public void shutdown() {}
```

Comment:
was it intended to be part of this PR? How is it related to interceptor?

Reply:
As the comment says, this will call shutdown on all the interceptors.

Reply:
ok